optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/SingleBenchmarkResult.java

```diff
@@ -60,6 +61,7 @@
     private Score score = null;
     private long timeMillisSpent = -1L;
     private long calculateCount = -1L;
+    private EnvironmentMode environmentMode = null;
```

Comment:
The EnvironmentMode is set in the solver config, so 7 single benchmarks belonging to the same solver benchmark will always have the same environmentMode, so this field is denormalized.